CSS: move block-level and inline-level pages from HTML to CSS (#25891)
mdn/content: files/en-us/learn/html/introduction_to_html/getting_started/index.md

The **tags have to open and close in a way that they are inside or outside one another**. With the kind of overlap in the example above, the browser has to guess at your intent. This kind of guessing can result in unexpected results.

> ### Element content categgories

Reviewer: Typo.
```suggestion
### Element content categories
```

> Most [HTML](/en-US/docs/Web/HTML) elements are a member of one or more **content categories** — these categories group elements that share common characteristics. This is a loose grouping (it doesn't actually create a relationship among elements of these categories), but they help define and describe the categories' shared behavior and their associated rules, especially when you come upon their intricate details. It's also possible for elements to not be a member of _any_ of these categories.

Reviewer 2: ```suggestion
Most [HTML](/en-US/docs/Web/HTML) elements are a member of one or more **content categories** — these categories group elements that share common characteristics. This is a loose grouping; it doesn't create a relationship among elements of these categories. These groupings help define and describe the categories' shared behavior and associated rules, especially when you encounter their intricate details. It's also possible for elements to not be a member of _any_ of these categories.
```

There are three types of content categories:

- Main content categories, which describe common rules shared by many elements.
- Form-related content categories, which describe rules common to form-related elements.

> - Specific content categories, which describe rare categories shared only by a few elements, sometimes only in a specific context.

Reviewer 2: we shouldn't have a header and then have a "go elsewhere to see the guts of this topic". The second half of this section is more confusing than instructive. We can 1) revert, 2) delete, or 3) move some content from Content categories here.

Author: I chose option 2 (delete). Let's not duplicate the info in two documents. Now, the hyper link has been attached to the term `Content categories`.

> For more info refer [Content categories](/en-US/docs/Web/HTML/Content_categories) page.

Reviewer: Grammar.
```suggestion
For more information refer to [Content categories](/en-US/docs/Web/HTML/Content_categories) page.
```

The **tags have to open and close in a way that they are inside or outside one another**. With the kind of overlap in the example above, the browser has to guess at your intent. This kind of guessing can result in unexpected results.

### Element content categories

> Most [HTML](/en-US/docs/Web/HTML) elements are a member of one or more [**Content categories**](/en-US/docs/Web/HTML/Content_categories) — these categories group elements that share common characteristics. This is a loose grouping; it doesn't create a relationship among elements of these categories. These groupings help define and describe the categories' shared behavior and associated rules, especially when you encounter their intricate details. It's also possible for elements to not be a member of _any_ of these categories.

Author: > this new version I think is beyond the scope of newbie content

You are right. I've removed the section.
